unit/java_bytecode/java_bytecode_parse_generics/WildcardGenericFunctions.java

```diff
@@ -1,54 +1,43 @@
 public class WildcardGenericFunctions
 {
   // Test a wild card generic type
-  public static void processSimpleGeneric(SimpleGeneric<?> x) {
-    assert(x.t.equals(null));
+  public static void processSimpleGeneric(Generic<?> x)
+  {
+
   }
 
   // Test a wildcard generic bound by an interface
-  public static void processUpperBoundInterfaceGeneric(SimpleGeneric<? extends BasicInterface> x) {
-    assert(x.t.getX() == 4);
+  public static void processUpperBoundInterfaceGeneric(Generic<? extends
+  Interface> x)
```

Comment:
Looks like your editor has done some auto-formatting here - could you make the line wrapping more readable? Maybe break after the `(` and then indent?